src/components/EntityInfo/EntityMenu.vue

```diff
@@ -300,6 +321,20 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
     return !!GetFeatureFlag('supported-amalgamation-entities').includes(this.getLegalType)
   }
 
+  get showConsentAmalgamateOut (): boolean {
```

Comment:
Thoughts in moving these to the store Sev? I see they're exactly the same across both files.

Reply:
I considered this and decided against it -- while they are the same now, the rules for what displays in the user menu versus the staff menu could differ. (Also the FF code will eventually be removed and the logic could be moved back into the template.)

Reply:
Makes sense. All right, thanks! Resolving.